### DISCUSSION TITLE
URL Not Found, Moved repo from Org to Personal

### DISCUSSION THREAD

**Discussion by ozayr** [codemagic-ci-cd/codemagic-docs#2361] [Q&A]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Flutter (Dart)

### Steps to reproduce

transfer repo from Org to personal

### Expected results

Code magic correctly picks up change and updates the repo URL. I had moved this repo from personal to the org and the update url worked perfectly now when moving back its not updating

### Actual results

`Cannot update repository https://api.github.com/repos/Organisation/repositoryName
Repository was not found`

 however when I click the link below the app name on the app settings page it takes me to the correct repo ie the repo in my personal account

**Comment by icarusdust** [comment]
Hi @ozayr, I do not see any issues when updating transferred repo URLs. The following steps are what I am doing right now, please let me know if something is different from what you are following:

 1. Go to GitHub repo settings, transfer the app to a GitHub Organisation
 2. After successfully transferring the app, open the repo in my Codemagic account
 3. Go to repository settings and click the Update repository URL button to complete the process. 

<img width="449" alt="Screenshot 2024-02-09 at 14 07 07" src="https://github.com/codemagic-ci-cd/codemagic-docs/assets/43378371/6234889d-f37d-497b-96b4-b795c69e35c2">

**Reply by icarusdust** [reply]
And I am transferring the app back and forth with no issues.

**Comment by icarusdust** [comment]
I am closing the thread due to inactivity. Let me know if you have any questions by replying to this thread or opening a new one.

**Reply by Mirshodbek** [reply]
Hi, I seem to have a problem that it can't find repositories. But we haven't changed the repository link at all. A couple of days ago I built the build without any problems, but suddenly it stopped building. And then there's this problem. Can you please tell me how to solve this problem? Thanks

<img width="1728" alt="Снимок экрана 2025-01-25 в 10 34 48" src="https://github.com/user-attachments/assets/d88b4998-1d2c-4b79-9d83-cefe1e13dc7d" />
Cannot access the repository
Not Found (404), the requested resource could not be found https://gitlab.com/api/v4/projects/50760924/repository/branches?per_page=100

<img width="1728" alt="Снимок экрана 2025-01-25 в 10 35 02" src="https://github.com/user-attachments/assets/9f27a229-34ad-4cad-8f39-71bb2a236327" />
Cannot update repository https://gitlab.com/api/v4/projects/50760924
Repository was not found
